README.md

```diff
@@ -60,67 +64,22 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
      orchestrator	https://parodos-dev.github.io/orchestrator-helm-chart
      ```
 
-   - **Manual Chart Deployment**\
-      By cloning the repository and navigating to the charts directory, you'll have access to the raw chart files and can customize them to fit your specific requirements.
-     ```console
-     git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
-     cd orchestrator-helm-chart/charts
-     ```
-
 2. Deploy the PostgreSQL reference implementation for persistence support in SonataFlow following these [instructions](https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/postgresql/README.md)
 
-3. Set `GITHUB_TOKEN` environment variable
-
-   ```console
-   export GITHUB_TOKEN=<github token>
-   ```
-
-4. Generate a random alphanumeric string of 24 characters to be used in the `BACKEND_SECRET`:
-
-   ```console
-   export BACKEND_SECRET=$(mktemp -u XXXXXXXXXXXXXXXXXXXXXXX)
-   ```
-
-5. Create a namespace for the Orchestrator solution:
+3. Create a namespace for the Orchestrator solution:
```

Comment:
nit: we can use `1.` everywhere, so no need to update the number list

Reply:
Nice trick 😄 . Done!